Source/Request.swift

```diff
@@ -57,25 +57,23 @@ public protocol RequestRetrier {
 
 // MARK: -
 
+protocol TaskConvertible {
```

Comment:
Okay, so after a long investigation, I think we're good to keep this internal. There's nothing about the `TaskConvertible` that needs to be exposed publicly. We don't have any issues subclassing `SessionManager` or `Request` or any of the subclasses. The `Request` initializer is also internal so I think we're good.

In the worst case scenario, we can always bump the ACL to public with only a MINOR version bump.